### PR TITLE
LGA 1829 out of scope diagnosis assign alternative help succeeds

### PR DESCRIPTION
**Pull request by jogilder**
Further steps for user journey p6 and refactoring

Created common functionality to deal with clicking buttons - allows for stale element exception.

Removed unused import statements and updates assert_header_on_page function to allow for more than one h1 element.

Changed I am taken to the "{type_of_user}" case details page as I think it was using the wrong step at the end.

Refactored "inscope" and "outofscope" methods so use same step.

Updated the steps at start of the @in_scope_debt_court to match other scenarios.

### CODE REVIEW

File: behave/features/alternative_help.feature [hunk 1]
```diff
@@ -15,4 +15,7 @@ Scenario: Create user and out of scope case
     AND I select the diagnosis <category> and click next <number> times
     | category                                                | number |
     | Crime                                                   | 2      |
-    THEN I get an OUTOFSCOPE decision
\ No newline at end of file
+    THEN I get an OUTOFSCOPE decision
```
Comment (psweeting1) [inline behave/features/alternative_help.feature:18]: `THEN` and other keywords are in capitals, other feature files aren't. My IDE complains if it's in caps. Does it cause any issues for you?

File: behave/features/constants.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import os
 
+MAX_TRIES = 2
```
Comment (said-moj) [inline behave/features/constants.py:3]: @jogilder do we still need this constant?

File: behave/helper/helper_base.py [hunk 3]
```diff
@@ -2,8 +2,9 @@
 from selenium.webdriver.common.by import By
 from selenium.webdriver.support.wait import WebDriverWait
 from selenium.webdriver.support import expected_conditions as EC
+from selenium.common.exceptions import StaleElementReferenceException
 from helper.backend import Backend
-from constants import CALL_CENTRE_ZONE
+from constants import CALL_CENTRE_ZONE, MAX_TRIES
```
Comment (said-moj) [inline behave/helper/helper_base.py:7]: @jogilder do we still need this constant?

File: behave/features/alternative_help.feature (hunk 1 above)
Comment (jogilder) [reply behave/features/alternative_help.feature]: It doesn't matter whether they are capitalised or not in terms of running the tests. I prefer lower case but others seem to like upper.

File: behave/features/steps/alternative_help.py [hunk 4]
```diff
@@ -31,3 +32,12 @@ def step_impl(context):
     context.execute_steps(u'''
         Then I will see the users details
     ''')
+
+
+@when(u'I select \'Assign Alternative Help\'')
```
[the remark this answers is not in the document]
Comment (jogilder) [reply behave/features/steps/alternative_help.py:37]: This is the only one that selects alternative help but it does raise the question of whether we should have wrappers for finding an `a` etc but then we would need wrappers for find a tag by class, by id etc so may not be any benefit.

File: behave/features/constants.py (hunk 2 above)
Comment (jogilder) [reply behave/features/constants.py:3]: Nope, not any more, I will remove it.

**Pull request merged**